unity-renderer/Assets/Scripts/MainScripts/DCL/WorldRuntime/KernelCommunication/IKernelCommunication.cs

```diff
@@ -0,0 +1,17 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.PlayerLoop;
+
+public class IKernelCommunication
+{
+    public IKernelCommunication()
+    {
+        
+    }
+
+    public virtual void Update()
```

Comment:
I'm not sure if we should expose an `Update()` method in the interface, as it would be used by a single inheritor. Maybe it's better to just declare an empty interface and use `CoroutineStarter.Start()` to implement the update in the WSS communication controller.